Files/UserControls/Widgets/RecentFiles.xaml

```diff
@@ -102,33 +103,41 @@
                                     Margin="0,0,0,0"
                                     VerticalAlignment="Stretch">
                                     <FontIcon
+                                        x:Name="FolderImg"
+                                        x:Load="{x:Bind FolderImg}"
+                                        x:Phase="2"
                                         FontFamily="{StaticResource FluentUIGlyphs}"
                                         FontSize="24"
                                         Foreground="#ffe793"
-                                        Glyph="&#xEA55;"
-                                        Visibility="{x:Bind FolderImg}" />
+                                        Glyph="&#xEA55;" />
                                     <FontIcon
+                                        x:Name="EmptyImg"
+                                        x:Load="{x:Bind EmptyImgVis}"
+                                        x:Phase="2"
                                         FontFamily="{StaticResource FluentUIGlyphs}"
                                         FontSize="24"
-                                        Glyph="&#xEA00;"
-                                        Visibility="{x:Bind EmptyImgVis}" />
+                                        Glyph="&#xEA00;" />
                                     <Image
+                                        x:Name="FileImg"
                                         Width="24"
                                         Height="24"
+                                        x:Load="{x:Bind FileIconVis}"
+                                        x:Phase="2"
                                         Source="{x:Bind FileImg}"
-                                        Stretch="UniformToFill"
-                                        Visibility="{x:Bind FileIconVis}" />
+                                        Stretch="UniformToFill" />
                                 </Grid>
                                 <TextBlock
                                     Grid.Column="1"
                                     VerticalAlignment="Center"
+                                    x:Phase="1"
```

Comment:
Any element without  x:Phase is considered "Phase 0". So you can probably remove the phase from TextBlock and decrease the phase on Images to "1".